src/app/visa/c.spec.ts

```diff
@@ -0,0 +1,630 @@
+                    'foreign_work_related_qualification',
+                ])
+            })
+
+            it('has worked on an advanced project in a growth field', () => {
+                const checklist = simulationWithCriteriaC([
+                    specialOf({ kind: 'advanced_project_growth_field' }),
+                ])
+
+                const { matches, points } = calculatePoints(checklist)
+
+                expect(points).toBe(10)
+                expect(matches.map(m => m.id).sort()).toEqual([
+                    'advanced_project_growth_field',
+                ])
+            })
+
+            it('completed training conducted by JICA as part of Innovative Asia Project', () => {
+                const checklist = simulationWithCriteriaC([
+                    specialOf({
+                        kind:
+                            'completed_training_conducted_by_jica_innovative_asia_project',
+                    }),
+                ])
+
+                const { matches, points } = calculatePoints(checklist)
+
+                expect(points).toBe(5)
+                expect(matches.map(m => m.id).sort()).toEqual([
+                    'completed_training_conducted_by_jica_innovative_asia_project',
+                ])
+            })
+
+            it('participates in investment business', () => {
+                const checklist = simulationWithCriteriaC([
+                    specialOf({ kind: 'investment_management_business' }),
+                ])
+
+                const { matches, points } = calculatePoints(checklist)
+
+                expect(points).toBe(10)
+                expect(matches.map(m => m.id).sort()).toEqual([
+                    'investment_management_business',
+                ])
+            })
+            it('qualifies for all special criteria', () => {
+                const checklist = simulationWithCriteriaC([
+                    specialOf({ kind: 'rnd_exceeds_three_percent' }),
+                    specialOf({ kind: 'foreign_work_related_qualification' }),
+                    specialOf({ kind: 'advanced_project_growth_field' }),
+                    specialOf({
+                        kind:
+                            'completed_training_conducted_by_jica_innovative_asia_project',
+                    }),
+                    specialOf({ kind: 'investment_management_business' }),
+
+                ])
+
+                const { matches, points } = calculatePoints(checklist)
+
+                expect(points).toBe(35)
+                expect(matches.map(m => m.id).sort()).toEqual([
+                    'advanced_project_growth_field',
+                    'completed_training_conducted_by_jica_innovative_asia_project',
+                    'foreign_work_related_qualification',
+                    'investment_management_business',
+                    'rnd_exceeds_three_percent',
+                ])
+            })
+        })
+
+        describe('contracting organization', () => {
```

Comment:
Can this be moved to be inside the `special` describe block, at the top of all of them? Having the tests in the same order of the actual points sheet makes it easier to review/update as the form changes.

Reply:
Some others are also out of order compared to the form, can these be fixed together?